cpp/tests/CMakeLists.txt

```
@@ -348,6 +348,15 @@ set(DEVICE_TABLE_TEST_SRC
 
 ConfigureTest(DEVICE_TABLE_TEST "${DEVICE_TABLE_TEST_SRC}")
 
+###################################################################################################
+# - sorted-merge tests -------------------------------------------------------------------------------------
+
+set(MERGE_TEST_SRC
+    "${CMAKE_CURRENT_SOURCE_DIR}/merge/merge-test.cu"
+    "${CMAKE_CURRENT_SOURCE_DIR}/utilities/nvcategory_utils.cu")
```

This shouldn't be necessary because `nvcategory_utils.cu` is already linked into `cudftestutil` lib. 

```suggestion
    "${CMAKE_CURRENT_SOURCE_DIR}/utilities/nvcategory_utils.cu")
```